tests/pytests/data/example_app.measurement

```diff
@@ -1 +1 @@
-�������ݒ��c�u����uԝ��47�
```

Comment:
if this file is generated during tests, you can add it to .gitignore

Reply:
OK, this one made sense and seemed to work when implemented. I took the opportunity to clean-up more generated files by adding them to `.gitignore`.

Will re-test to make sure things still run in CI ...

Reply:
@rgerganov -- I had to re-instate the existing files in `tests/pytests/data` dir. I could not remove them as some existing pytests are written to work off of these checked-in files. (CI job failed miserably otherwise.)

So the only change arising from your comment is some cleanup in `.gitignore`. As that was simple enough, I squashed all post-review commit into the main one. You can just take a re-look at this one .gitignore file.